[2024-09-02T16:12:53.233+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T16:12:53.244+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:12:52.178347+00:00 [queued]>
[2024-09-02T16:12:53.250+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:12:52.178347+00:00 [queued]>
[2024-09-02T16:12:53.251+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T16:12:53.259+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): first_task> on 2024-09-02 16:12:52.178347+00:00
[2024-09-02T16:12:53.266+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3896) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T16:12:53.268+0000] {standard_task_runner.py:72} INFO - Started process 3897 to run task
[2024-09-02T16:12:53.266+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'manual__2024-09-02T16:12:52.178347+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpskhihdn9']
[2024-09-02T16:12:53.270+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask first_task
[2024-09-02T16:12:53.317+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:12:52.178347+00:00 [running]> on host cedbab6b7256
[2024-09-02T16:12:53.375+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T16:12:52.178347+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T16:12:52.178347+00:00'
[2024-09-02T16:12:53.377+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T16:12:53.398+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T16:12:53.399+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-02T16:12:53.476+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T16:12:53.485+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-02T16:12:53.488+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T16:12:53.516+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T16:12:53.519+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=manual__2024-09-02T16:12:52.178347+00:00, execution_date=20240902T161252, start_date=20240902T161253, end_date=20240902T161253
[2024-09-02T16:12:53.574+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T16:12:53.596+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-02T16:12:53.599+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
